examples/pre-made-lammps/README.md

```diff
@@ -0,0 +1,16 @@
+## Pebble Centroid Map Dataset v1.0.0
+
+This dataset contains pebble centroid coordinates for the fuel pebbles in a
+pebble-bed reactor.
+
+Each dataset is paired with the input file that made it.  The list of dataset groups are as follows:
+
+### Generic HTGR SMR 1
+All particles, both type 1 and type 2, are 3[cm] or 0.03 [m] in radius.  Note that the units of the coordinates and accompanying input file are in meters.
```

Comment:
What are type 1 and type 2 pebbles?

Reply:
Generally speaking, if I refer to a 'type', I'm referring to the particle type in LAMMPS.  You use types to distinguish particles with different parameters, like size or density.  In the case of this specific dataset, the type was used to make them different colors.